src/getting-started/README.md

```diff
@@ -6,34 +6,28 @@ sidebar_label: Introduction
 
 ## Introduction
 
-Before you begin, read [Personium Architecture](../user_guide/001_Personium_Architecture.md).
+This document is an introductory article for those who are new to Personium. The purpose of this document is to familiarize you with the basics of Personium so that you can use it.
 
-Figure
+## Prerequisite knowledge
 
-Whole steps are:
+If you have not read [Personium architecture](../user_guide/001_Personium_Architecture.md), it is recommended to read it as a common prerequisite knowledge of Personium.
```

Comment:
Although optional, reading [Personium architecture](../user_guide/001_Personium_Architecture.md) before proceeding is strongly recommended.